python/tvm/script/builder/builder.py

```diff
@@ -0,0 +1,22 @@
+from tvm._ffi import register_object as _register_object
+
+from tvm.runtime import Object
+
+from . import _ffi_api
+
+
+@_register_object("script.builder.Builder")
+class Builder(Object):
+    def __init__(self) -> None:
+        self.__init_handle_by_constructor__(
+            _ffi_api.Builder
+        )
+
+    def __enter__(self) -> None:
+        _ffi_api.EnterBuilder(self)
+
+    def __exit__(self, ptype, value, trace) -> None:
+        _ffi_api.ExitBuilder(self)
+
+    def currentBuilder(self):
+        return _ffi_api.CurrentBuilder(self)
```

Comment:
Let's be consistent in naming: https://github.com/apache/tvm/blob/main/python/tvm/target/target.py#L137-L150